createSpotifyPlaylist.py

```diff
@@ -0,0 +1,256 @@
+"""
+This module is used to create spotify playlist from other playlists. This is possible by using the
+spotify api which allows us to search and retrive data. We use the spotify playlists to search for
+playlist with a provided search query. All the playlists that are found will be search for its most
+popular songs. Those songs will then be added to a new spotify playlist which will be craeted here.
+This module could be run on its own to create playlist for whoever wants it or could also be run
+using the module `twitter.py` thats in this project. What this will do is check the users twitter
+timeline and create playlists by checking @mentions and using those as search querys. This will
+require both the spotify and twitter api.
+"""
+import os
+import time
+import spotipy
+from spotipy.oauth2 import SpotifyOAuth
+
+
+
+class CreateSpotifyPlaylist:
+    """
+    A class used to communicate with the Spotify api.
+
+    Attributes
+    ----------
+    popular_tracks : list
+        A list of popular tracks on spotify
+
+    Methods
+    -------
+    get_spotify_playlists(self, query:str)
+        Searches spotify for playlists
+
+    get_popular_songs(self, spotify_playlist_id:str):
+        Gets most popular songs from spotify playlist
+
+    create_spotify_playlist(self, playlist_name:str, for_user:str)
+        Creates a new spotify playlists using a name and a users name
+
+    add_tracks_to_playlist(self, playlist_id:str)
+        Adds song to a spotify playlists
+    """
+
+    def __init__(self):
+        """
+        Init function
```

Comment:
`Init function` is obvious, so this line is not required

Reply:
I removed this line